.github/ISSUE_TEMPLATE/new_product_suggestion.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Product Suggestion
+about: Suggest a new product to add into endoflife.date
+title: ''
+labels: 'new-product'
+assignees: ''
+
+---
+
+`Full and Short Name of Suggested Product`
+
+Let us know its full long name and if exist its short name ( ie PowerShell , pwsh )
```

Comment:
"i.e." means "that is", and "e.g." means "for example". It's better to use the full words and avoid the Latin initialisms.

```suggestion
Let us know the long, full name and, if it has one, the short name (for example, PowerShell and pwsh).
```

Reply:
done